README.rst

```diff
@@ -9,6 +9,7 @@ Introduction
 ------------
 
 This is a micro-frontend application responsible for the display of an authenticated user's order history and receipt page.
+Additionally, it also displays subscription history as of July 2023.
```

Comment:
Suggestion - 
`This is a micro-frontend application responsible for the display of an authenticated user's single purchase order history and subscription history.
Additional work has been done and is in progress to enable receipt pages in this application, currently hosted in ecommerce.`

Reply:
Context my suggestion - not related to your work for subscriptions - technically the receipt page is not in this MFE 100% yet, the work has been paused and will resume later.

Reply:
sure, makes sense